packages/node/src/index.ts

```diff
@@ -70,6 +70,7 @@ export default class LogtoClient extends BaseClient {
       return {
         isAuthenticated,
         claims,
+        userInfo: await this._fetchUserInfo(fetchUserInfo),
```

Comment:
How about checking and calling `fetchUserInfo` in prior? e.g.
```ts
const userInfo = conditional(fetchUserInfo && (await this.fetchUserInfo()));
```
This way, you don't have to declare a private `_fetchUserInfo` method

Reply:
updated